src/main/java/core/specialevents/ExperienceEventPredictionAnalyzer.java

```diff
@@ -0,0 +1,108 @@
+package core.specialevents;
+
+import core.model.player.IMatchRoleID;
+import core.model.player.MatchRoleID;
+import core.model.player.Player;
+import module.lineup.Lineup;
+
+import java.util.List;
+import java.util.Vector;
+
+import static java.lang.Math.abs;
+
+
+/*
+ Experienced Forward Scores
+
+ If your forward has great experience compared to the average experience of the opponents, and also high enough scoring
+ to outwit the keeper, you might end up with a goal.
+
+ Example: If your forward has 10 experience against an average of 8 and his scoring is 13 against a 15 goalkeeper, your
+ chances are 33% to score. But if the experience was at 15, your chances increase to 38%.
+
+ Inexperienced Defender
+
+ If your defender has low experience compared to the average experience of the opposing team, an opposing player may be
+ able to win the ball and then get a crack at scoring.
+
+ Example: A defender with 10 experience and 15 defending against a forward with 13 scoring (against a 15 keeper) and an
+ average experience of 12, will give the opponent 20% to score. But if you increase your defender experience to 12, you
+ will drop this to 16%.
+
+ */
+public class ExperienceEventPredictionAnalyzer implements ISpecialEventPredictionAnalyzer {
+
+    private SpecialEventsPredictionManager.Analyse analyse;
+
+    @Override
+    public void analyzePosition(SpecialEventsPredictionManager.Analyse analyse, MatchRoleID position) {
+
+        this.analyse = analyse;
+
+        //double experienceAverage = getExperienceAverage();
+        int id = position.getSpielerId();
+        Player p = analyse.getPlayer(id);
+        SpecialEventsPrediction se=null;
+        switch (position.getId()){
+            /*
+            case IMatchRoleID.leftCentralDefender:
```

Comment:
what about this comments? If this code is not used, maybe we can remove it, wdyt?

Reply:
thanks for your comments-  and you are right, at the end this comments could be removed. i didn't remove it already, because this is the prediction where i simplified the specifications a little. the commented code was the approach that handled the specification as it is. i would like to discuss if you would agree with my simplification. in case of inexperience defender i would have the problem to estimate the opponents scoring strength.